### DISCUSSION TITLE
Ability to add a Note Card without creating an Issue

### DISCUSSION THREAD

**Discussion by faminchik** [community/community#7098] [Projects and Issues] [answered]
Hello,

Consider, please, the possibility to create a simple Note Card on a Board (as it works for **Projects**) without creating an Issue in the project repository.

Thanks

**Comment by azenMatt** [comment, marked as answer]
Hi @faminchik - would you mind talking a little more about what you are trying to achieve here that the new 'draft issues' (which let you add in simple text to the board) don't allow for?

**Reply by faminchik** [reply]
Hi @azenMatt.

Sorry for the misunderstanding.

Yes, If just add a simple text (without assigning a repository), it will look like a simple Note Card.
And only after assigning the repository, it's converted to an Issue

I'll close this discussion, thanks!

**Reply by zdk123** [reply]
I disagree that draft issues solves this -  e.g. they cannot be assigned to a specific user, or be used to link multiple related issues together, etc.

**Reply by Bikappa** [reply]
I also don't think the draft issues cover some use-cases like:
- managing internal work on a public repository, confidential info might be made public simply assigning the card to a repo
- they don't allow easy and accountable collaboration since colleagues can't add comments until turned into an issue
